appservice/src/tree/FolderTreeItem.ts

```diff
@@ -39,7 +42,7 @@ export class FolderTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
-        const kuduClient: KuduClient = await this.root.client.getKuduClient();
+        const kuduClient: KuduClient = await this.client.getKuduClient();
         let response: IKuduItemResponse;
         try {
             response = <IKuduItemResponse><unknown>(await kuduClient.vfs.getItemWithHttpOperationResponse(this.path)).response;
```

Comment:
Since you had the files node more or less working for Trial Apps, I'm assuming that the kuduClient API call (on line 45) worked to retrieve the files?

Reply:
Yes, I was able to get the KuduClient, and retrieve files for the trial app.

The prototype code for that is here: